.github/workflows/ci.yml

```diff
@@ -35,6 +35,34 @@ jobs:
           command: ${{matrix.command}}
           args: "${{matrix.command == 'fmt' && '--all -- --check' || '-- -D warnings'}}"
 
+  brew:
+    name: brew
+    runs-on: macos-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        rust: [stable]
+        include:
+          - {version: 1.8}
+          - {version: '1.10'}
+          - {version: 1.12}
+          - {version: 1.12, mpi: true}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with: {submodules: true}
+      - name: Install Rust (${{matrix.rust}})
+        uses: actions-rs/toolchain@v1
+        with: {toolchain: '${{matrix.rust}}', profile: minimal, override: true}
+      - name: Install HDF5 (${{matrix.hdf5}})
```

Comment:
`matrix.version`?

Reply:
Will fix. Feel free to cancel the actions